com.unity.template.hdrp-blank/Packages/com.unity.template.hdrp-blank/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [17.0.6] - 2024-10-09
 
-## Changed
```

Comment:
Fixing https://unity-ci.cds.internal.unity3d.com/job/45731045/results?r=45731046

Reply:
EDIT: actual fix seems to bump the template, see below. It was probably missed in https://github.com/Unity-Technologies/Graphics/pull/8104